runtime/bcutil/attrlookup.h

```diff
@@ -168,7 +168,7 @@ lookupKnownAttribute (register const char *str, register unsigned int len)
 #line 71 "attrlookup.gperf"
       {"NestMembers", CFR_ATTRIBUTE_NestMembers, CFR_ATTRIBUTE_NestMembers},
 #line 72 "attrlookup.gperf"
-      {"MemberOfNest", CFR_ATTRIBUTE_MemberOfNest, CFR_ATTRIBUTE_MemberOfNest},
+      {"NestHost", CFR_ATTRIBUTE_NestHost, CFR_ATTRIBUTE_NestHost},
```

Comment:
Was `gperf` run to regenerate this file after the input changed?

Reply:
No, I forgot - is it better to download + install + rerun gperf for a whole new file or to just directly edit one of the entries in the `asso_values` array in attrlookup.h to minimize code changes?

Reply:
gperf is available on most linux machines.  As gperf generates a perfect hash based on the inputs, its always better to regenerate than hand-hack the file.

Reply:
Updated using gperf.

May need to wait until I return to other laptop to fix the ip-validation - I've signed the ECA for both accounts and signed off as `Talia McCormick <Talia.McCormick@ibm.com>` on both but can't validate that email address using my current computer.